config/environments/test.rb

```diff
@@ -13,15 +15,18 @@
   # just for the purpose of running a single test. If you are using a tool that
   # preloads Rails for running tests, you may have to set it to true.
   # NOTE: set it to false would result in CI tests to timeout
-  config.eager_load = true
+  config.eager_load = false
```

Comment:
Explain the changes in this file?

Reply:
this is actually because the comment just up there explains that the `eager_load` should be set as `false`. And now that we have moved to `Zeitwerk`, which strongly promotes the efficiency in doing the loading, I tried to change it back to what it should be (`false`) and now nothing breaks so I decided to change it like that

Though, I should have deleted that NOTE so as not to confuse future dev